src/Nancy/Json/Simple/NancySerializationStrategy.cs

```diff
@@ -72,8 +72,9 @@ protected override string MapClrMemberNameToJsonFieldName(string clrPropertyName
         /// </summary>
         /// <param name="value">The object to deserialize</param>
         /// <param name="type">The type of object to deserialize</param>
+        /// <param name="dateTimeStyles">The <see cref="DateTimeStyles"/> ton convert <see cref="DateTime"/> objects</param>
         /// <returns>A instance of <paramref name="type" /> deserialized from <paramref name="value"/></returns>
-        public override object DeserializeObject(object value, Type type)
+        public override object DeserializeObject(object value, Type type, DateTimeStyles dateTimeStyles)

```

Comment:
I can't as it overrides the PocoJsonSerializerStrategy DeserializeObject method and that class has no constructor